src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md

```diff
@@ -49,82 +49,84 @@ During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud
 
 ## FAQs
 
-### What does limited access refer to regarding the availability of CockroachDB {{ site.data.products.dedicated }} on Azure?
+The following sections provide more details about CockroachDB {{ site.data.products.dedicated }} on Azure.
 
-CockroachDB {{ site.data.products.dedicated }} on Azure is fully managed, just like CockroachDB {{ site.data.products.dedicated }} on GCP or AWS. During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), your CockroachDB {{ site.data.products.cloud }} organization must be enrolled before you can start using CockroachDB {{ site.data.products.dedicated }} on Azure.
+### What is Limited Access and how does it relate to CockroachDB {{ site.data.products.dedicated }} on Azure?
+
+CockroachDB {{ site.data.products.dedicated }} on Azure is fully managed, like on GCP or AWS. During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), your {{ site.data.products.db }} organization must be enrolled before you can deploy CockroachDB {{ site.data.products.dedicated }} clusters on Azure.
 
 The clusters created during this period are recommended for proof-of-concept and testing, and are not suitable for production. The [CockroachDB {{ site.data.products.cloud }} Service Level Agreement (SLA)](https://cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/) is not applicable to Azure clusters during limited access. Azure clusters are excluded from premium support agreements during limited access, and technical support is only available during business hours.
 
-### Can we create multi-region dedicated clusters on Azure?
+### Are multi-region clusters supported?
 
 Not yet. During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), a cluster can be created only in a single region, and a cluster must have three or more nodes. A cluster's nodes are automatically placed in different [availability zones](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-overview) to ensure resiliency to failure of a single availability zone.
 
-### Is it possible to horizontally scale a dedicated cluster on Azure?
+### Is horizontal scaling supported?
 
 Not yet.
 
-### What Azure regions can we choose to create the dedicated clusters?
+### What Azure regions can we choose?
 
 You can create a cluster in `eastus2` (US East Coast - Virginia) or `westeurope` (Netherlands) during the limited access period. Contact your account team to express interest in other regions.
```

Comment:
I think the list here will need to be expanded with the added regions.

Reply:
Done

Reply:
Done